client/components/Logo.tsx

```diff
@@ -1,19 +1,20 @@
 import React from 'react';
-import { useNavigate } from 'react-router-dom';
```

Comment:
Is there a plan to use this component in the future or should it just be deleted?

Reply:
Deleted.